chef_master/source/perform_compliance_scan.rst

```diff
@@ -189,9 +172,10 @@ A truncated version of the bootstrap output is shown below:
    localhost [2017-05-17T21:11:11+00:00] INFO: Report handlers complete
    localhost Chef Client finished, 0/2 resources updated in 18 seconds
 
+.. note:: The ``audit`` cookbook also requires a time synchronization policy to be in place, such as Network Time Protocol (NTP).
+
 View the results in Chef Automate
 -------------------------------------------------------
-
 When you go back to your Chef Automate UI under the **Compliance** tab, the **Reporting** dashboard should be visible. By alternating between **Node Status** and **Profile Status** views, you can view the scan results depending on which view is most important to you. The following shows some of the scan results on the bootstrapped node, "node1-ubuntu".
```

Comment:
We shouldn't be instructing users to bootstrap vagrant instances or setting the node name. I would remove the `, "node1-ubuntu"` bit.

Reply:
ok